pkg/util/exec/testdata/test_trap_sigterm.sh

```diff
@@ -14,8 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-printf >&1 '%s' "running"
+echo running
```

Comment:
not sure why this was more complicated than echo before.. changed it because it feels less distracting.